### PR TITLE
Remove using gae-specific db url entries from Vault

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
## Context

_Say a few words about the bigger picture beyond this PR. How do your changes improve life for our users? Link directly to the jira ticket (just type "DDP-..." and it'll automatically link) to help your teammates quickly get up to speed on the motivation for these changes._

## Checklist

- [ ] I have labeled the type of changes involved using the `C-*` labels.
- [ ] I have assessed potential risks and labeled using the `R-*` labels.
- [ ] I have considered error handling and alerts, and added `L-*` labels as needed.
- [ ] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [ ] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

**Review by zyme — APPROVED** [review]

**Review by zyme — DISMISSED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -308,6 +314,40 @@ commands:
...
+            API_BASE_PATH="<< parameters.config_repo_path >>/contents"
+            FILE_PATH="<< parameters.config_file_path >>/${ENVIRONMENT}/dispatch.yaml"
+            FILE_URL="${API_BASE_URL}${API_BASE_PATH}${FILE_PATH}"
+            BRANCH="<< parameters.config_branch >>"
+            FULL_URL="${FILE_URL}?ref=${BRANCH}"
+            curl /
+              -H 'Accept:application/vnd.github.v3.raw' /
+              "$FULL_URL" /
+              -o dispatch.yaml
+            cat dispatch.yaml
```
Comment (MocanaAtBroad) [inline .circleci/config.yml:355]: Where we get the new dispatch.yaml file from the angular repo.

File: pepper-apis/config/application.conf.ctmpl [hunk 2]
```diff
@@ -11,13 +10,8 @@
...
-    {{if eq $gae "true"}}
-        "dbUrl": "{{$conf.Data.gaeDbUrl}}",
-        "housekeepingDbUrl": "{{$conf.Data.gaeHousekeepingDbUrl}}",
-    {{else}}
-        "dbUrl": "{{$conf.Data.dbUrl}}",
-        "housekeepingDbUrl": "{{$conf.Data.housekeepingDbUrl}}",
-    {{end}}
+    "dbUrl": "{{$conf.Data.dbUrl}}",
+    "housekeepingDbUrl": "{{$conf.Data.housekeepingDbUrl}}",
     "defaultTimezone": "{{$conf.Data.defaultTimezone}}",
```
Comment (MocanaAtBroad) [inline pepper-apis/config/application.conf.ctmpl:15]: if else be gone.

File: .circleci/config.yml [hunk 3]
```diff
@@ -633,6 +674,7 @@ jobs:
           jar_base_name: Housekeeping
       - deploy-jar:
           jar_base_name: Housekeeping
+      - update-dispatch-rules
```
Comment (MocanaAtBroad) [inline .circleci/config.yml:688]: We deploy we update rules.

File: .circleci/config.yml (hunk 1 above)
Comment (zyme) [reply .circleci/config.yml:355]: Niiice

**Review by zyme — APPROVED** [review]

**Pull request merged**